internal/internal.go

```diff
@@ -15,7 +15,13 @@ import (
 	"strings"
 )
 
+var configDir string
+
 func ConfigDir() (string, error) {
+	if configDir != "" {
```

Comment:
I'd vote for us using @rakyll's suggestion e.g
```go
if configDir := os.Getenv("GOPS_CONFIG_DIR"); configDir != "" {
       return configDir, nil
}
``` 
This prevents us from have to mock with the global but it also enables the user to set up their config once e.g in the .bashrc file and set it to anywhere